mesh/ncmesh.hpp

```diff
@@ -64,7 +64,11 @@ struct CoarseFineTransformations
    /// Fine element positions in their parents.
    Array<Embedding> embeddings;
 
-   void GetCoarseToFineMap(const Mesh &fine_mesh,
+   /// The number of nonghost elements on the original mesh. This is only
+   /// needed in the case of derefinement.
+   int orig_elements;
```

Comment:
This is the problem, is that it's not known in the use case which motivated this. It would be possible to require the application code to keep track of this in the calling context and pass it in, but that seems less preferable to having the library keep track of this so it is automatic.

Reply:
I can't find any "old_dof_table" variable or close variation in the code.

Reply:
Sorry, it's called `old_elem_dof` in `RefinementOperator`. The number of rows is the number of elements in the previous mesh.